drive/driveimpl/drive_test.go

```diff
@@ -30,14 +31,19 @@ import (
 const (
 	domain = `test$%domain.com`
 
-	remote1 = `rem ote$%1`
-	remote2 = `_rem ote$%2`
-	share11 = `sha re$%11`
-	share12 = `_sha re$%12`
-	file111 = `fi le$%111.txt`
+	remote1 = `rem ote$%<>1`
```

Comment:
Drive-by: we weren't properly escaping paths for inclusion in XML, this tests for that.